packages/api/orm/entity/Clin.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseEntity } from "./BaseEntity";
+import { Column, Entity, ManyToOne } from "typeorm";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Clin extends BaseEntity {
+  @Column({
+    type: String,
+    nullable: false,
+    length: 4,
+    comment: "contract line item number from task order, 0001 through 9999",
+  })
+  clinNumber: string;
+
+  @Column({
+    type: String,
+    nullable: false,
+    comment: "indefinite-delivery, indefinite-quantity CLIN specific to JWCC",
+  })
+  idiqClin: string;
+
+  // decimal(17,2) allows tens of trillions
+  @Column({
+    type: "decimal",
+    nullable: false,
+    precision: 17,
+    scale: 2,
+    default: 0.0,
+    comment: "full amount requested by customer from USG for project or contract",
+  })
+  totalClinValue: number;
+
+  @Column({
+    type: "decimal",
+    nullable: false,
+    precision: 17,
+    scale: 2,
+    default: 0.0,
+    comment: "amount already disbursed by USG for project or contract to be spent during CLIN POP",
+  })
+  obligatedFunds: number;
```

Comment:
Postgres does have a `money` type that TypeORM supports. Is there a benefit to decimal vs money? https://www.postgresql.org/docs/13/datatype-money.html

Reply:
Hmm.  We'd been using decimal/number in the API spec and TypeScript to this point.  I hadn't considered the money type.

Reply:
Yeah as long as we can easily convert to and from money & Number between the DB and API then that would be a nice option.

Reply:
Tried it.  Went smoothly.  Didn't even need to adjust my insert.

[This reference](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_money) includes the following, to which we generally conform.  This should be ok for our use case.
> When should you?
If you're only working in a single currency, aren't dealing with fractional cents and are only doing addition and subtraction then money might be the right thing.

Reply:
Changed in a0c0142b756741ad4379072e0f6efe80f57efa25